pkg/crypto/tinkcrypto/primitive/secp256k1/secp256k1_signer_key_manager_test.go

```diff
@@ -278,14 +278,6 @@ func TestPublicKeyDataWithInvalidInput(t *testing.T) {
 
 	_, err = pkm.PublicKeyData(serializedKey)
 	require.Error(t, err, "expect an error when input is a modified serialized key")
-
-	// nil
-	_, err = pkm.PublicKeyData(nil)
```

Comment:
No longer returns an error with the updated Tink version (pub key bytes in returned struct are just empty now)

Reply:
you can remove the whole test function since the main test is not applicable anymore

Reply:
@Baha-sk You mean remove `TestPublicKeyDataWithInvalidInput`? Are the other test cases in it not applicable?

Reply:
yes, the whole test function's purpose was to test empty/nil cases, it seems Tink now panics on purpose in these cases... you can probably recover the panic here and make sure it does actually panic instead of removing these calls? that would be a good test case.

Reply:
There are some other test cases in this function still that work the same as before above the code I removed). For these two particular test cases I removed, they just don't return an error anymore (i.e. are not an invalid test case anymore)

Reply:
Good point re: catching the panic - I'll update the other places in this PR where I removed those tests that panic to catch it

Reply:
> There are some other test cases in this function still that work the same as before (above the code I removed, in this same function). For these two particular test cases I removed, they just don't return an error anymore (i.e. are not an invalid test case anymore)

I think you can use `require.Panics()` in this case, without having to set a value.

Reply:
These two cases don't panic - they're not an error or panic